npm-shrinkwrap.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "fh-fhc",
-  "version": "1.0.2-BUILD-NUMBER",
+  "version": "2.0.0-BUILD-NUMBER",
```

Comment:
Are we breaking the previous forms FHC APIs with this change? Should this definitely be a major bump? 
I'm also thinking yes, but just wanted to raise the question!